NitroxModel/Logger/Log.cs

```diff
@@ -23,6 +23,12 @@ public static void SetLevel(LogLevel level)
             Write("Log level set to " + Log.level);
         }
         
+        // For chat logging
+        public static void ChatMsg(string name, string msg)
+        {
+            Console.WriteLine("{0}: {1}", name, msg);
```

Comment:
I don't know if we need this.  We already have Log.Write().  You can pass in an already formatted string.